src/core/generator/folderGenerator.ts

```diff
@@ -157,14 +175,31 @@ const disableIconsByPack = (
   });
 };
 
+/**
+ * Set the icon definitions in the manifest.
+ *
+ * @param manifest - The manifest object to be updated.
+ * @param config - The configuration object for the icons.
+ * @param icon - The icon to be set in the manifest.
+ * @param appendix - The appendix to be added to the icon name.
+ * @returns The updated manifest object with the icon definitions.
+ */
 const setIconDefinitions = (
   manifest: Manifest,
   config: Config,
-  icon: FolderIcon | DefaultIcon
+  icon: FolderIcon | DefaultIcon,
+  appendix: string = ''
```

Comment:
The parameter is never used. I was wondering why we should add it then. From my point of view there's no reason to make this adjustable via parameter at the moment.